### PR TITLE
fixing terraform issues reported by fmichaelobrien

### LINKED ISSUE #1: Example fortigate-tutorial-gcp deployment
Good afternoon team,. I will be reviewing this your implementation.  Thank you for the excellent work.  Keeping this issue open for initial discussion on config/deploy/use items.
/Michael

### PR DESCRIPTION
**Pull request by bartekmo**
- added documentation on required APIs (for new projects)
- PAYG deployments (without BYOL license files) are now handled smoother
- fix the reference issue for the newly created subnetworks

### CODE REVIEW

**Comment by fmichaelobrien** [comment]
Thanks for the quick turnaround, I was planning on entering a couple prs as I adjusted and edited the scripts.  Saving me some time, I will review this pr later tonight

**Review by jamie-pate — APPROVED** [review]

**Review by Joel-Cripps — APPROVED** [review]

File: terraform/day0/README.md [hunk 1]
```diff
@@ -25,6 +33,8 @@ The `fgcp-ha-ap-lb` module takes a list of subnet names as its `subnets` argumen
 - port3 (nic2) - FGCP hertbeat/sync interface
 - port4 (nic3) - dedicated management interface
 
+Make sure your project quota is high enough to accommodate for the new VPC networks.
```
Comment (fmichaelobrien) [inline terraform/day0/README.md:36]: I'll writup the procedure to get your per-project quota increased from 5 (default) to 10 - usually the 2 min AI bot will kick in depending on how you write the quota request (paid services only)

File: terraform/modules/sample-networks/main.tf [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   count                   = length(var.networks)
-  name                    = "${var.prefix}-${var.networks[count.index]}-vpc"
+  name                    = "${var.prefix}vpc-${var.networks[count.index]}"
   auto_create_subnetworks = false
 }
 
 resource "google_compute_subnetwork" "subnets" {
   count         = length(var.networks)
-  name          = "${var.prefix}-${var.networks[count.index]}-sb"
+  name          = "${var.prefix}sb-${var.networks[count.index]}"
```
Comment (fmichaelobrien) [inline terraform/modules/sample-networks/main.tf:9]: good catch

**Review by fmichaelobrien — APPROVED** [review]
LGTM
I'll put in some more details on the gcloud version later
I will retest the terraform deploy over this week as well - thanks for the changes

**Review by jvhoof — APPROVED** [review]

**Pull request merged**